# Request
string path_name
---
# Response
bool success
string message